### PR TITLE
Adding a simple run command that will run local commands on a docker …

### PR DESCRIPTION
**Pull request by trorabaugh**
Adding a simple run command that will run local commands on a docker container of choice. By default this will allow you to run things like wc or other default docker container commands.

### CODE REVIEW

**Comment by dorsha** [comment]
@anara123 can you review this one?

File: Scripts/script-RunDockerCommand.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+commonfields:
+  id: 9ffdaa61-efdd-47e6-8a88-ea86c949274f
```
Comment (anara123) [inline Scripts/script-RunDockerCommand.yml]: change to RunDockerCommand

File: Scripts/script-RunDockerCommand.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  version: 2
```
Comment (anara123) [inline Scripts/script-RunDockerCommand.yml]: version -1

File: Scripts/script-RunDockerCommand.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+name: RunDockerCommand
+script: |-
+  # The command demisto.command() holds the command sent from the user.
```
Comment (anara123) [inline Scripts/script-RunDockerCommand.yml]: delete the comment

File: Scripts/script-RunDockerCommand.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          output = subprocess.check_output(cmdarg, shell=True)
+      except subprocess.CalledProcessError as e:
+          output = e.output
+      ret.append(output)
+  else:
+      ret.append(subprocess.check_output([cmdarg, sysargs]))
+
+
+  ec = {'Command': cmdarg, 'Results': ret[0]}
+  demisto.results({'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': ret, 'HumanReadable': ret[0], 'EntryContext': {'CommandResults': ec}})
```
Comment (anara123) [inline Scripts/script-RunDockerCommand.yml]: prettify the json 
`{'ContentsFormat': formats['json'],'Type': entryTypes['note'],'Contents': hosts_json,
      'HumanReadable':res,'EntryContext': ec}` => 
```
{
    'Type': entryTypes['note'],
    'ContentsFormat': formats['json'],
    'Contents': hosts_json,
    'HumanReadable':res,
    'ReadableContentsFormat': formats['markdown'],
    'EntryContext': ec
}

```

**Comment by anara123** [comment]
@trorabaugh is it have to run on native python or also from inside the docker too?

**Comment by Itay4** [comment]
@trorabaugh What is the status of this?

File: Scripts/script-RunDockerCommand.yml (hunk 1 above)
Comment (bakatzir) [reply Scripts/script-RunDockerCommand.yml]: done

File: Scripts/script-RunDockerCommand.yml (hunk 2 above)
Comment (bakatzir) [reply Scripts/script-RunDockerCommand.yml]: done

File: Scripts/script-RunDockerCommand.yml (hunk 3 above)
Comment (bakatzir) [reply Scripts/script-RunDockerCommand.yml]: done

File: Scripts/script-RunDockerCommand.yml (hunk 4 above)
Comment (bakatzir) [reply Scripts/script-RunDockerCommand.yml]: done

**Pull request merged**